frog-utils/src/dashboards/progress.js

```diff
@@ -50,24 +50,27 @@ export const LineChart = ({
 );
 
 const TIMEWINDOW = 5;
+let timingData;
 
 const Viewer = TimedComponent((props: Object) => {
   const { data, instances, activity, timeNow } = props;
 
-  const numWindow = Math.ceil(
-    (timeNow - activity.actualStartingTime) / 1000 / TIMEWINDOW
-  );
-  const timingData = [[0, 0, 0]];
-  const factor = 100 / Object.keys(instances).length;
-  for (let i = 0, j = -1; i <= numWindow; i += 1) {
-    if (i * TIMEWINDOW === (data['timing'][j + 1] || [0])[0]) {
-      j += 1;
+  if (activity.actualClosingTime === undefined) {
```

Comment:
I think a good way to do is would be to do:

```

const ViewerWithTime = ({ data, instances, activity, timeNow }) => {
    ...
    return ( ... )
}

const Viewer = (props) =>
    props.activity.actualClosingTime 
        ? <ViewerWithTime {...props} timeNow={activity.actualClosingTime} />
        : <TimedComponent(ViewerWithTime, 5000) {...props} /> 
```

Reply:
I changed the suggestion for how to include the activityclosingtime